app/services/album.js

```diff
@@ -25,3 +26,30 @@ exports.photos = albumId =>
     .catch(() => {
       throw errors.externalApiError('Error in external API');
     });
+
+exports.buy = async (albumId, currentUser) => {
+  try {
+    const response = await axios.get(`${apiUrl}/albums/${albumId}`);
```

Comment:
If this fails, the catch will log a database error and then simply forward the error as is, without wrapping it in our errors object.